calico_containers/calico_ctl/container.py

```diff
@@ -90,26 +92,26 @@ def container(arguments):
                     container_remove(arguments.get("<CONTAINER>"))
         if arguments.get("endpoint"):
             endpoint.endpoint_show(hostname,
-                                   ORCHESTRATOR_ID,
+                                   None,
```

Comment:
IIRC, the endpoint lookup does a recursive lookup based on the maximum number of complete path segments that can be constructed and then does a linear search to match the remaining segments.

Upshot, if you remove the orchestrator like you are doing here, we'll end up enumerating all of the workloads even though you're actually passing it in.